datalad_next/runners/iter_subproc.py

```diff
@@ -20,6 +22,7 @@ def iter_subproc(
     *,
     input: Iterable[bytes] | None = None,
     chunk_size: int = COPY_BUFSIZE,
+    cwd: Path | None = None,
```

Comment:
Since `Popen` supports `Path`, `bytes`, and `str` from Python 3.8 on Posix and Windows, let's put them in here.
```suggestion
    cwd: Path | bytes | str | None = None,
```